src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    logged. Each entry must be a string representing the name of the space
+    to monitor.
+
+    Example:
+
+    `spaces: [bands, singers]`, only the events of `bands` and `singers` spaces
+    are logged.
+]])
+
+M['audit_log.to'] = 'Enable audit logging and define the log location.'
+
+M['audit_log.syslog'] = format_text([[
+    This module allows configuring the system logger (syslog) for audit
+    logs in Tarantool. It provides options for specifying the syslog server,
+    facility, and identity for logging messages.
+]])
+
+M['audit_log.syslog.facility'] = format_text([[
+    Specify a system logger keyword that tells syslogd where to send the
+    message. You can enable logging to a system logger using the
+    `audit_log.to` option.
+]])
+
+M['audit_log.syslog.identity'] = format_text([[
+    Specify an application name to show in logs. You can enable logging
+    to a system logger using the `audit_log.to` option.
+]])
+
+M['audit_log.syslog.server'] = format_text([[
+    Set a location for the syslog server. It can be a Unix socket path
+    starting with "unix:" or an ipv4 port number. You can enable logging
+    to a system logger using the `audit_log.to` option.
+]])
+
+-- }}} audit_log configuration
+
+-- {{{ compat configuration
+
+M['compat'] = format_text([[
+    The `compat` section defines values of the compat module options.
+]])
+
+M['compat.binary_data_decoding'] = format_text([[
+    Define how to store binary data fields in Lua after decoding:
+
+    - `new`: as varbinary objects
+    - `old`: as plain strings
+]])
+
+M['compat.box_cfg_replication_sync_timeout'] = format_text([[
+    Set a default replication sync timeout:
+
+    - `new`: 0
+    - `old`: 300 seconds
+]])
+
+M['compat.box_consider_system_spaces_synchronous'] = format_text([[
+    Controls whether to consider system spaces synchronous when the
+    synchronous queue is claimed, regardless of the user-provided `is_sync`
+    option. Either enables synchronous replication for system spaces when the
+    synchronous queue is claimed, overriding the user-provided 'is_sync' space
+    option, or falls back to the user-provided 'is_sync' space option.
+]])
+
+M['compat.box_error_serialize_verbose'] = format_text([[
+    Set the verbosity of error objects serialization:
+
+    - `new`: serialize the error message together with other potentially
+    useful fields
+    - `old`: serialize only the error message
+]])
+
+M['compat.box_error_unpack_type_and_code'] = format_text([[
+    Whether to show error fields in `box.error.unpack()`:
+
+    - `new`: do not show `base_type` and `custom_type` fields; do not show the
+    `code` field if it is 0. Note that `base_type` is still accessible for an
+    error object
+    - `old`: show all fields
+]])
+
+M['compat.box_info_cluster_meaning'] = format_text([[
+    Define the behavior of `box.info.cluster`:
+
+    - `new`: show the entire cluster
+    - `old`: show the current replica set
+]])
```

Comment:
AFAIR, it also affects `box.info.replicaset` (present only in the new behavior).

I would clarify it like so:

```markdown
- `new`: `box.info.cluster` shows info about the entire cluster,
         `box.info.replicaset` shows info about the replica set
- `old`: `box.info.cluster` shows info about the replica set
```

Reply:
Ok. Let's do it this way.